lib/core/api.js

```diff
@@ -489,6 +489,19 @@ api.upload.toPlatform = function(data, sessionInfo, progress, groupId, cb, uploa
       //generate and post the upload metadata
       var now = new Date();
 
+      if (rollbar) {
+        var events = _.filter(data, function(event) {
+          var year = parseInt(event.time.substring(0,4));
+          var timestamp = sundial.parseFromFormat(event.time).getTime();
+
+          return year < 2006 || timestamp > (now.getTime() + (24 * 60 * sundial.MIN_TO_MSEC));
```

Comment:
I think this `now.getTime()` could be moved to a variable above this function so it's executed once for the whole set of events instead of per-event.